### PR TITLE
hw1 - Basic HTML and CSS

### PR DESCRIPTION
**Pull request by NOcode-developer**
1. Switched to the main branch and performed a git pull.
2. Created a new branch "1-html_NOcode-developer" from the main branch.
3. Created a directory named "GamesProject" in the local repository.
4. Inside "GamesProject," created an "index.html" file.
5. Created a "style.css" file and linked it to "index.html."
6. Added semantic markup to the content.
7. Added styles for the blocks and text.

### CODE REVIEW

File: homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        </header>
+        <main>
+            <article class="about-me">
+                <section class="img-about">
+                    <img src="./assets/img/image-about1.svg" alt="image about us">
+                </section>
+                <section class="text-about">
+                    <h1>About me</h1>
+                    <p>
+                        Hi! My name is [Text] and I'm a Junior Frontend
```
Comment (vladyslav-yermolin-moc) [inline homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html:27]: Don't you like to add your name here?

File: homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  width: 100%;
+}
+
+.text-about {
+  width: 50%;
+  max-width: none;
+  display: block;
+}
+
+.text-about>*+* {
```
Comment (vladyslav-yermolin-moc) [inline homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css:45]: This selector is hard to read and understand. Do you need it? Maybe better to add this margin to paragraphs?

File: homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  line-height: 32px;
+  margin: 0;
+}
+
+a {
+  color: #EF4934;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 32px;
+  text-decoration-line: underline;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css:70]: All links are underlined by default, so you can skip this.
image

File: homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                            target="_blank">'Frontend for beginners' from Masters Academy in
+                            2023</a>.
+                    </p>
+                    <p>
+                        This is a social project from MOCG company where I got
+                        an opportunity to work with Frontend mentors and to
+                        create my own small project for the portfolio.
+                    </p>
+                    <p>
+                        You can contact me via [Links to Socials Nets] and/or
```
Comment (vladyslav-yermolin-moc) [inline homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html:44]: Same here: you can add links to your Social nets and GitHub now. Or later, on the final stage of the project.

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html (hunk 1 above)
Comment (NOcode-developer) [reply homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html]: oops... I missed that

File: homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css (hunk 2 above)
Comment (NOcode-developer) [reply homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css]: I fixed it by adding margin-top

File: homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css (hunk 3 above)
Comment (NOcode-developer) [reply homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css]: removed

File: homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html (hunk 4 above)
Comment (NOcode-developer) [reply homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html]: I thought it was just plain text; it was my inattention. I've fixed it and added the links

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Pull request merged**